cypress/pages/SiteLogIn.spec.js

```diff
@@ -0,0 +1,28 @@
+export class Site {
+
+  open() {
+    const userAndPass = `${Cypress.env('siteAuthUserName')}:${Cypress.env('siteAuthPassword')}`;
+    return cy.visit(`https://${userAndPass}@qauto.forstudy.space/`);
+  }
+
+  userLogIn() {
+    cy.request('POST', '/api/auth/signin', {
+      'email': Cypress.env('user'),
+      'password': Cypress.env('password'),
+      'remember': false,
+    });
+  }
+
+  profileTabOpen() {
+    const userAndPass = `${Cypress.env('siteAuthUserName')}:${Cypress.env('siteAuthPassword')}`;
+    return cy.visit(`https://${userAndPass}@qauto.forstudy.space/panel/profile/`);
```

Comment:
```suggestion
    return cy.visit(`https://qauto.forstudy.space/panel/profile/`);
```
На цьому етапі юзер зазвичай вже буде залогінений

Reply:
тест падає на цьому степі, якщо зробити так як ви написали
![Uploading Screenshot 2023-12-14 at 19.35.49.png…]()